crm_phonecall_planner/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+* Incompatible with ``crm_phonecall_summary_predefined`` addon.
```

Comment:
Nice to have: Maybe you could solve this open roadmap point before merging. With PR #327 extending crm_phonecall is easier. Maybe you have an idea to eliminate this problem.

Reply:
Sorry, i don't see how to use these improvements to remove this incompatibility.